docs/template-vm.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: linux-vm-template
+  annotations:
+    description: "OCP kubevirt linux, template"
+    tags: "kubevirt,ocp,template,linux"
+    miq.github.io/kubevirt-is-vm-template: true
+objects:
+- apiVersion: kubevirt.io/v1alpha1
+  kind: OfflineVirtualMachine
+  metadata:
+    name: ${NAME}
+  spec:
+    template:
+      spec:
+        domain:
+          resources:
+            requests:
+              memory: ${MEMORY}
+              cpu:
```

Comment:
that needs to go under `spec.domain.cpu.cores`. `resources.requests.cpu` is about cgroup assignment

Reply:
Ack, thanks for catching this.